packages/playground/src/context/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactElement, ReactNode, useReducer, useState } from "react";
+import { Action, defaultState, State, reducer } from "~/redux";
```

Comment:
Should not be named "~/redux", reducer should bne an own file in Context folder.